try:
    pass
except:
    pass
finally:
    match name_0:
        case False:
            pass
name_2.name_4

# output:
# python: Python/flowgraph.c:511: no_redundant_jumps: Assertion `0' failed.
